admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+==========================
+Configuring Memory Caching
+==========================
+
+You can significantly improve your ownCloud server performance with memory 
+caching, where frequently-requested objects are stored in memory for faster 
+retrieval. There are two types of caches to use: PHP opcache, and data caching 
```

Comment:
If PHP 5.4 is not supported anymore i would agree but still this version doesn't ship OPCache and so there are stuff mixed here. Just think that not everyone is tech-savy and understand the difference here.